src/main/scala/latis/input/HapiAdapter.scala

```diff
@@ -135,4 +133,15 @@ object HapiAdapter {
     }
   }
 
+  /**
+   * Given a FDM model, create a list of variable names
+   * for the HAPI "parameters" query parameter.
+   * Note that vector components with IDs: "foo._1", "foo._2"
+   * will be reduced to a single "foo" parameter.
+   */
+  def buildParameterList(model: DataType): List[String] =
+    model.getScalars
+         .drop(1)    //drop implicit time variable
```

Comment:
Not a big deal, but I thought our style was to only indent with two spaces on long chains of functions like this, instead of lining up the dots.

Reply:
Good question. I tend to align dots if the first isn't in "too far". scalafmt does as you suggest and I can live with that. Any other thoughts?
There's also the option:
```scala
model
  .getScalars
  .drop(1)
```
It looks like scalafmt will do that if the first dot is in "too far".